lib/data/external/GCP/GcpService.js

```diff
@@ -63,6 +63,13 @@ const GCP = Service.defineService('gcp', ['2017-11-01'], {
         }
     },
 
+    // Implemented APIs
+    // Bucket API
+    getBucket(params, callback) {
+        return this.listObjects(params, callback);
```

Comment:
just checking, where is `listObjects` defined?

Reply:
listObjects is defined in the service definition file, `lib/data/external/GCP/gcp-2017-11-01.api.json`